pkg/handler/admin/admin.go

```diff
@@ -29,19 +28,25 @@ func (base *Controller) GetUsers(c *gin.Context) {
 
 }
 
-// this returns the mined images of all users
+// Get          godoc
+// @Summary     this returns the mined images of all users
+// @Description this returns the mined images of all users
+// @Tags        Mined Images
+// @Produce     json
+// @Success     200  {object} []model.MinedImage
+// @Router      /admin/mined-images [get]
 func (base *Controller) GetAllMinedImages(c *gin.Context) {
 
-	secretKey := config.GetConfig().Server.Secret
-	token := utility.ExtractToken(c)
+	//secretKey := config.GetConfig().Server.Secret
+	//token := utility.ExtractToken(c)
 
-	_, err := utility.GetKey("id", token, secretKey)
+	//_, err := utility.GetKey("id", token, secretKey)
 
-	if err != nil {
-		rd := utility.BuildErrorResponse(http.StatusUnauthorized, "failed", "could not verify token", nil, gin.H{"error": err.Error()})
-		c.JSON(http.StatusUnauthorized, rd)
-		return
-	}
+	//if err != nil {
+	//rd := utility.BuildErrorResponse(http.StatusUnauthorized, "failed", "could not verify token", nil, gin.H{"error": err.Error()})
+	//c.JSON(http.StatusUnauthorized, rd)
+	//return
+	//}
```

Comment:
why did you comment this out?
because it's used to verify the auth token.

Reply:
i'm closing this pr and pushing a new one